Wangscape/logging/Level.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+namespace logging
+{
+
+
+enum class Level
+{
+    Debug,
+    Info,
+    Warning,
+    Error
```

Comment:
So... you're suggesting that `Debug` level should be for a developers and the lowest level for messages to an user should be `Info`. Do I understand it correctly? With a bit different design we could even have a different logger for our needs. But...

I'm thinking about bug reports - these "developer debugging messages" would be really helpful I think. What about this? Do you agree?